test/image-blueprints-bootc/layer3-periodic/group1/rhel96-bootc-source-ai-model-serving.containerfile

```diff
@@ -19,12 +19,9 @@ COPY ./bootc-images/$USHIFT_RPM_REPO_NAME.repo ./bootc-images/microshift-fast-da
 # vim-common for xxd utility
 RUN dnf repoinfo --enabled && \
     dnf install -y \
-        {{ range (env.Getenv "MICROSHIFT_OPTIONAL_RPMS" | strings.Split " ") -}}
-        "{{ . }}-{{ env.Getenv "SOURCE_VERSION" }}" \
-        {{ end -}}
-        {{ range (env.Getenv "MICROSHIFT_X86_64_RPMS" | strings.Split " ") -}}
-        "{{ . }}-{{ env.Getenv "SOURCE_VERSION" }}" \
-        {{ end -}}
+        "microshift-release-info-{{ env.Getenv "SOURCE_VERSION" }}" \
```

Comment:
I think `microshift-release-info` is not needed because it's already present in `rhel96-bootc-source:latest`, wdyt?